### PR TITLE
created database and table notes

### PR DESCRIPTION
**Pull request by bennyho94**
A database has been created with the name notes_db including a table notes_tb.
The table has the following columns:

- title with a varchar that consists of maximum 100 characters.
- note with a varchar that consists of maximum 200 characters.
- author with a varchar that consists of maximum 100 characters.
- the amount of characters has been set to 100 & 200 due to the fact that it's reasonable for a note. I believe that people will keep it short and simple.
- These 3 columns has also a NOT NULL which means that a value is always required.
- timestamp with DEFAULT CURRENT_TIMESTAMP that automatically initialize and updates the time.
- Assigned the title as the primary key

### CODE REVIEW

**Comment by iliastic** [comment]
Why no ID?

**Comment by bennyho94** [comment]
> Why no ID?

Due to the fact that ID is not mandatory to use. Hence, I've assigned the title as my primary key to be my unique field

File: notes.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+CREATE DATABASE notes_db;
```
Comment (ghost) [inline notes.sql]: add IF Not Exist

File: notes.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+CREATE TABLE notes_tb {
+    title varchar(100) NOT NULL,
+    note varchar(200) NOT NULL,
```
Comment (ghost) [inline notes.sql]: 200 is to small for the note

File: notes.sql (hunk 1 above)
Comment (bennyho94) [reply notes.sql]: Okay, thanks for your feedback!

File: notes.sql (hunk 2 above)
Comment (bennyho94) [reply notes.sql]: Haha it might be! Have modified it to 1000. Thanks!

**Pull request merged**